QUEST_JOBSTEP.tsv

```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
 QUEST_JOBSTEP_20150714_002870	Go to the Pardoner Master in Akmens Ridge.
-QUEST_JOBSTEP_20150714_002871	The Pardoner Master will let you skip the advancement assignment, but asked you to retrieve the money of lost indulgence. Find the lost money at Aqueduct Bridge Area.
-QUEST_JOBSTEP_20150714_002872	Defeat Red Triambulos at Apsimesti Crossroads.
-QUEST_JOBSTEP_20150714_002873	The Sadhu Master asked you to defeat the monsters that are threatening the villagers. Defeat Red Triambulos from Apsimesti Crossroads.
+QUEST_JOBSTEP_20150714_002871	The Pardoner Master will let you skip the advancement assignment, but asked you to retrieve the Lost Indulgence Offerings. Find the Lost Indulgence Offering at Aqueduct Bridge Area.
+QUEST_JOBSTEP_20150714_002872	Defeat Red Tree Ambulos at Apsimesti Crossroads.
+QUEST_JOBSTEP_20150714_002873	The Sadhu Master asked you to defeat the monsters that are threatening the villagers. Defeat Red Tree Ambulos from Apsimesti Crossroads.
 QUEST_JOBSTEP_20150714_002874	Talk to the Monk Master in the Goddess' Ancient Garden.
-QUEST_JOBSTEP_20150714_002875	The Tools Merchant wants hardened oil in exchange for information. Gather Hardened Oil from Puragis in Saknis Plains and give it to the Tools Merchant.
-QUEST_JOBSTEP_20150714_002876	go to the Sorcerer Master in Forest of Prayer.
-QUEST_JOBSTEP_20150714_002877	Rodolero Master told you that you are now a true Rodelero, but also told you that you can still improve. Defeat Colimencia at the garden of Green Family and find out what's lacking in you.
+QUEST_JOBSTEP_20150714_002875	The Tools Merchant wants Hardened Oil in exchange for information. Gather Hardened Oil from Puragis in Saknis Plains and give it to the Tools Merchant.
+QUEST_JOBSTEP_20150714_002876	Go to the Sorcerer Master in Forest of Prayer.
+QUEST_JOBSTEP_20150714_002877	Rodolero Master told you that you are now a true Rodelero, but also told you that you can still improve. Defeat Colimencia at the Greene Manor and find out what's lacking in you.
 QUEST_JOBSTEP_20150714_002878	Go to the Cataphract Master in Gateway of Great King.
 QUEST_JOBSTEP_20150714_002879	Go to the Thaumaturge Master in Gateway of Great King.
 QUEST_JOBSTEP_20150714_002880	The Thaumaturge Master wants to know whether you would be able to surpass the next level. Defeat Netherbovine that is making a mess in Poslinkis Forest.
 QUEST_JOBSTEP_20150714_002881	Masters in Akmens Ridge are chasing the monster causing mayhem and disappearing. Defeat Netherbovine that appeared in Poslinkis Forest.
-QUEST_JOBSTEP_20150714_002882	Meet the Sorcerer Master in Forst of Prayers.
-QUEST_JOBSTEP_20150714_002883	The Sorcerer Master told you that you should not stay still since the monsters are going crazy these days. Go to Goddess' Ancient Gardenes and defeat Necrobenter.
+QUEST_JOBSTEP_20150714_002882	Meet the Sorcerer Master in Forest of Prayers.
```

Comment:
Forest of Prayers -> Forest of Prayer

Also, remove the period. (Could you do a simple CTRL + F to change this along with the removal of the period, I see it in some more lines)